src/Management/test/Endpoint.Test/ManagementPort/ManagementEndpointServedOnDifferentPortTest.cs

```diff
@@ -98,6 +98,37 @@ public async Task AspNetDefaultPort_AlternateManagementPortConfigured_Accessible
         actuatorResponse.StatusCode.Should().Be(HttpStatusCode.OK);
     }
 
+    [Fact]
+    public async Task AspNetDefaultPort_AlternateManagementPortConfigured_AccessibleOnCfInstancePorts()
+    {
+        const string managementPort = "8090";
+        const string externalTlsProxyPort = "8998";
+        const string internalTlsProxyPort = "8999";
+
+        using var instancePorts = new EnvironmentVariableScope("CF_INSTANCE_PORTS",
+            $"[{{\"external\":40041,\"internal\":{managementPort},\"external_tls_proxy\":{externalTlsProxyPort},\"internal_tls_proxy\":{internalTlsProxyPort}}}]");
```

Comment:
```suggestion
        using var instancePorts = new EnvironmentVariableScope("CF_INSTANCE_PORTS", $$"""
            [
              {
                "internal": {{managementPort}},
                "external_tls_proxy": {{externalTlsProxyPort}},
                "internal_tls_proxy": {{internalTlsProxyPort}}
              }
            ]
            """);
```

Aside from making it readable, removed the "external" property.

Reply:
I am fine with the change in formatting, but I wonder if we should leave "external" there to match actual values found on the platform (though that is only on Windows)